src/beachbot/ai/yolov5_torch_hub.py

```diff
@@ -99,18 +102,20 @@ def apply_model(self, inputs, confidence_threshold=0.2, units_percent=True):
                 width = res[i, 2] - res[i, 0]
                 height = res[i, 3] - res[i, 1]
                 if units_percent:
-                    left /= self.img_width
-                    top /= self.img_height
-                    width /= self.img_width
-                    height /= self.img_height
+                    # do not use model format, we use resolution of current input...
+                    # Pytorch performs probably fome fance rescaling of the image (to fit the yolo resolution) under the hood
+                    # left /= self.img_width
+                    # top /= self.img_height
+                    # width /= self.img_width
+                    # height /= self.img_height
```

Comment:
You can keep this in a separate branch if you want it for your own notes. In a branch that won't be merged or used otherwise.